### DISCUSSION TITLE
Get the two Lines between two points on the polygon

### DISCUSSION THREAD

**Discussion by mziadd** [locationtech/jts#899] [Q&A] [answered]
Suppose we have a polygon I want to the lines between Point F and Point C, Is there any Class That finds them easily?

For Example, I want the lines 
1. F->E->D->C
2. F->A->B->C

Or if there are any hints/ ideas.

Many thanks

![Polygon_vertices](https://user-images.githubusercontent.com/24648104/180269012-690a796f-d1e8-47a2-b127-936940f545d8.jpg)

**Comment by FObermaier** [comment, marked as answer]
Have a look at the `NetTopologySuite.LinearReferencing` namespace especially `LocationIndexOfPoint` and `ExtractLineByLocation` classes.

**Comment by mziadd** [comment]
Thanks i find a solution using turf slice line
